packages/github/src/context.ts

```diff
@@ -44,7 +44,7 @@ export class Context {
 
     return {
       ...this.repo,
-      number: (payload.issue || payload.pullRequest || payload).number
+      number: (payload.issue || payload.pull_request || payload).number
```

Comment:
@thboop a pull request is not an issue?

Reply:
The issue linked to this PR indicates the \"linked issue\" is desired.

It looks like the change above returns the pull request number, not the inked issue number.

Reply:
discussed offline. before only PR events works, this change adds PR comment and PR review comments